development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -98,10 +98,11 @@ Start a terminal, go to the root dir of the engine source code and type:
 
 ::
 
-    scons -j8 platform=linuxbsd
+    scons -j`nproc` platform=linuxbsd
 
 A good rule of thumb for the ``-j`` (*jobs*) flag, is to have at least as many
 threads compiling Godot as you have cores in your CPU, if not one or two more.
+```nproc``` can be used to automatically use the number of cores your system has.
```

Comment:
```suggestion
``$(nproc)`` can be used to automatically use the number of cores your system has.
```

Reply:
I updated the PR to explain how to use all cores automatically on each OS instead of using `nproc` directly.